### PR TITLE
"Proof read" chapter 0 and 1

### PR DESCRIPTION
**Pull request by FFFluoride**
I originally was only going to fix typos but I just couldn't help myself. I showed some restraint but I would totally be open to some changes if you disagree with some of the changes made in this PR.

I also realised that proof reading is boring and time consuming lol. This means that I won't be doing this again soon, maybe when I actually have time.

Again, thanks for writing this book

### CODE REVIEW

**Comment by FFFluoride** [comment]
Well, I guess I'll just submit the PR

**Comment by abuseofnotation** [comment]
Thanks a lot for the effort. Yes, it's super hard and time-consuming, I even don't know how I wrote this :)

**Comment by FFFluoride** [comment]
> Thanks a lot for the effort. Yes, it's super hard and time-consuming, I even don't know how I wrote this :)
You've just made my day, I really needed to hear that.

Tell me if you disagree with any of my decisions.

File: _chapters/00_about.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ Like we said, the fundaments of mathematics are the fundaments of thought. Categ
 
 The way we think and talk is based on intuition that develops naturally and is a very easy way to get our point across. However, that is part of the problem: sometimes intuition makes it *too easy* for us to say something that can be interpreted in many ways, some of which are wrong. For example, when I say that two things are equal, it would seem obvious to you what I meant, although it isn't obvious at all (how are they equal?, in what context?, etc.) 
 
-It is in these situations that people often resort to *diagrams* to refine their thoughts. Diagrams (even more than formulas) are ubiquitous in science and mathematics.
+It's in these situations that people often resort to *diagrams* to refine their thoughts. Diagrams (even more than formulas) are ubiquitous in science and mathematics.
```
Comment (abuseofnotation) [inline _chapters/00_about.md:110]: Why do you think the shortened form is better?

Comment (FFFluoride) [reply _chapters/00_about.md:110]: I thought that more concise was better but I can change that if you want

Comment (abuseofnotation) [reply _chapters/00_about.md:110]: OK, I thought I'd correct the things myself, but I will drop some comments instead, maybe you will get something out of the conversations

Comment (FFFluoride) [reply _chapters/00_about.md:110]: what conversation? what comments? I'm kind of confused.

**Review by abuseofnotation — COMMENTED** [review]
Sorry for the delay, I just found out that I had to submit the review to get my comments published.... (;

File: _chapters/00_about.md [hunk 2]
```diff
@@ -65,16 +65,16 @@ Although many people don't necessarily subscribe to this view of mathematics as
 
 There is nothing wrong with this approach, but mathematics is so much more than solving problems. It was the basis of a religious cult in ancient Greece (the Pythagoreans), it was seen by philosophers as means to understanding the laws which govern the universe. It was, and still is, a language which can allow for people with different cultural backgrounds to understand each other. And it is also art and a means of entertainment.
 
-Category theory embodies all these aspects of mathematics, so I think a very good grounds to writing a book where all of them shine --- a book that is based not on solving of problems, but on exploration of concepts and on seeking connections between them. A book that is, overall, pretty.
+Category theory embodies all these aspects of mathematics, so I think it's very good grounds to writing a book where all of them shine --- a book that isn't based on solving of problems, but exploring concepts and seeking connections between them. A book that is, overall, pretty.
```
[the remark this answers is not in the document]
Comment (FFFluoride) [reply _chapters/00_about.md:69]: Yeah, there's no problem with removing words, I can add the on back. However, do you want to revert it completely or just add the on?

**Comment by FFFluoride** [comment]
Honestly, I might just give more of the book a scan through but not scrutinize it so much (like I did here)

File: _chapters/00_about.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (abuseofnotation) [reply _chapters/00_about.md:69]: Do you find the construct "based not on ... but on ..." so bad?

[the remark this answers is not in the document]
Comment (abuseofnotation) [reply _chapters/00_about.md:69]: Do you find the construct "based not on ... but on ..." so bad?

[the remark this answers is not in the document]
Comment (FFFluoride) [reply _chapters/00_about.md:69]: No, but I don't like how what comes after it sounds like in my head, but maybe that's just me

[the remark this answers is not in the document]
Comment (abuseofnotation) [reply _chapters/00_about.md:69]: OK, let's leave it as is then.

**Comment by abuseofnotation** [comment]
Please do. I will include you in the acknowledgements :)

**Pull request merged**